picard/pluginmanager.py

```diff
@@ -273,8 +275,11 @@ def _load_plugin_from_directory(self, name, plugindir):
         if zip_importer:
             name = module_name
             if not zip_importer.find_module(name):
-                error = _("Failed loading zipped plugin %r from %r")
-                self.plugin_error(name, error, name, zipfilename)
+                error = _('Failed loading zipped plugin "%(plugin)s" from "%(filename)s"')
```

Comment:
It looks like this PR wasn't based on `master` but on [phw:weblate-unnamed-variables-check](https://github.com/phw/picard/tree/weblate-unnamed-variables-check)

Reply:
Indeed. Fixed now